addons/web/static/src/views/fields/domain/domain_field.js

```diff
@@ -21,29 +23,40 @@ export class DomainField extends Component {
         context: { type: Object, optional: true },
         editInDialog: { type: Boolean, optional: true },
         resModel: { type: String, optional: true },
+        isFoldable: { type: Boolean, optional: true },
     };
     static defaultProps = {
         editInDialog: false,
+        isFoldable: false,
     };
 
     setup() {
         this.orm = useService("orm");
+        this.getDomainTreeDescription = useGetDomainTreeDescription();
         this.addDialog = useOwnedDialogs();
 
         this.state = useState({
             isValid: null,
             recordCount: null,
+            folded: true,
```

Comment:
folded: this.props.isFoldable?

folded: true gives the impression that it is always folded